test/event-stream-test.ts

```diff
@@ -0,0 +1,98 @@
+import { expect, test } from "@playwright/test";
+import { createAppFixture, createFixture, js } from "./helpers/create-fixture.js";
+import type { AppFixture, Fixture } from "./helpers/create-fixture.js";
+import { PlaywrightFixture } from "./helpers/playwright-fixture.js";
+
+test.describe("check event stream", () => {
+  let fixture: Fixture;
+  let appFixture: AppFixture;
+
+  test.beforeAll(async () => {
+    fixture = await createFixture({
+      files: {
+        "src/routes/index.jsx": js`
+        import { createEffect, createSignal, onCleanup, Show } from "solid-js";
+        import server from "solid-start/server";
+        
+        function createEventStream({ url }, onMessage) {
+          createEffect(() => {
+            const eventSource = new EventSource(url);
+        
+            eventSource.addEventListener("chat", (event) => {
+              onMessage(event);
+            });
+        
+            onCleanup(() => eventSource.close());
+          });
+        }
+        
+        function eventStream(request, init) {
+          let stream = new ReadableStream({
+            start(controller) {
+              let encoder = new TextEncoder();
+              let send = (event, data) => {
+                controller.enqueue(encoder.encode("event: " + event + "\n"));
+                controller.enqueue(encoder.encode("data: " + data + "\n" + "\n"));
+              };
+              let cleanup = init(send);
+              let closed = false;
+              let close = () => {
+                if (closed) return;
+                cleanup();
+                closed = true;
+                request.signal.removeEventListener("abort", close);
+                controller.close();
+              };
+              request.signal.addEventListener("abort", close);
+              if (request.signal.aborted) {
+                close();
+                return;
+              }
+            },
+          });
+          return new Response(stream, {
+            headers: { "Content-Type": "text/event-stream" },
+          });
+        }
+        
+        export default () => {
```

Comment:
```suggestion
        export default function Page() {
```

Just for consistency with other tests